packages/cloudflare/src/api/kv-cache.ts

```diff
@@ -116,6 +117,8 @@ class Cache implements IncrementalCache {
       );
     } catch {
       throw new RecoverableError(`Failed to set cache [${key}]`);
+    } finally {
```

Comment:
looks weird to have a dependency here!

Reply:
from what i can tell, there isn't a way to know when a revalidation is complete from within the queue itself, so i put a call to it from the incremental cache instead.

two things come to mind:
1. remove the entry after the fetch call instead of after the incremental cache set
2. change the cache stuff in aws to invoke a new method on a queue after an incremental cache set

i can take a look at the second option. it sounds reasonable. just would be nice if we had a monorepo and i could do it in this pr.

Reply:
done the second option using https://github.com/opennextjs/opennextjs-aws/pull/734

Reply:
conico made a good point about chance of separate isolates handling requests... let me change to the first option.